packages/plugins/simple-vault/package.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
ignore this whole thing for now. Need to figure out how we will deal with CLIs and plugins before this plugin can be useful.

Reply:
maybe cherry pick it into another branch/PR?